pyriemann_qiskit/classification.py

```diff
@@ -922,6 +922,8 @@ def _predict_distances(self, X):
                 dist = self._process_sample(x)
                 dists.append(dist)
 
+        dists = np.concatenate(dists, axis=1)
```

Comment:
@qbarthelemy I do not understand. Did you run the code before the commit?
There is another run time error here?

Reply:
Solved.